api/realm.go

```diff
@@ -7,54 +7,56 @@ import (
 )
 
 const (
-	realmRootPath               = "/auth/admin/realms"
-	realmPath                   = realmRootPath + "/:realm"
-	realmCredentialRegistrators = realmPath + "/credential-registrators"
-	exportRealmPath             = "/auth/realms/:realm/export/realm"
+	// API Keycloak out-of-the-box
+	kcRealmRootPath               = "/auth/admin/realms"
+	kcRealmPath                   = kcRealmRootPath + "/:realm"
+	kcRealmCredentialRegistrators = kcRealmPath + "/credential-registrators"
+	// ABANDONNED ctExportRealmPath = "/auth/realms/:realm/export/realm"
```

Comment:
This could also be deleted with the below commented code, unless this is useful information